app/controllers/RequiredInformationController.scala

```diff
@@ -16,23 +16,20 @@
 
 package controllers
 
-import javax.inject.Inject
-
-import scala.concurrent.{ExecutionContext, Future}
-
-import play.api.i18n.{I18nSupport, MessagesApi}
-import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
-import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendBaseController
-
 import controllers.actions._
 import forms.RequiredInformationFormProvider
-import models.Mode
-import models.UserAnswers
+import models.{NormalMode, UserAnswers}
 import navigation.Navigator
```

Comment:
I presume IntelliJ is doing this, might just need to ensure it's using scalafmt and not the built in formatter

Reply:
hm i think so